### ISSUE TITLE
Pre-train model with CenterNet Detector

### ISSUE THREAD

**Issue by EthanHNguyen** [lkevinzc/dance#20]
Hi,

Sorry for bothering you, and thank you for the excellent work on DANCE! I am trying to better understand DANCE's improvements over DeepSnake, and I would like to reproduce more of DANCE's results. Do you have the COCO training script and pre-trained model for DANCE with CenterNet as the detector?

Thank you so much.

**Comment by lkevinzc** [comment]
Hi @EthanHNguyen , thank you for your interest!

CenterNet is used in the `snake` branch to compare with DeepSnake (implemented on top of their codebase), mainly on Cityscapes and SBD dataset.

For COCO dataset, my personal view is that stronger detector benefits, since the bbox score is kind of upper-bound the mask score. I mainly base on the Detectron2 and work on FCOS for this case.

One possible way to reproduce is that, based on the `snake` branch, you could run experiments on COCO (since DeepSnake's code also supports this). I think I have implement a version of DANCE in such setting, please have a check here for setup and run: https://github.com/lkevinzc/dance/tree/snake#get-started. 

Feel free to leave any follow-up comments here! Thanks! :)
